src/components/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId?: string;
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { tabs, selectedTabId } = props;
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
```suggestion
  const tabContent = tabs
    .find(tab => tab.id === selectedTabId)?.content || 'Please select a tab';
```

You can prepare content here to simplify render code.